etna/metrics/metrics.py

```diff
@@ -67,6 +69,78 @@ def greater_is_better(self) -> bool:
         """Whether higher metric value is better."""
         return False
 
+class RMSE(Metric):
+    """Root mean squared error metric with multi-segment computation support.
+
+    .. math::
+        RMSE(y\_true, y\_pred) = \\sqrt{\\frac{\\sum_{i=0}^{n-1}{(y\_true_i - y\_pred_i)^2}}{n}}
+
+    Notes
+    -----
+    You can read more about logic of multi-segment metrics in Metric docs.
+    """
+
+    def __init__(self, mode: str = MetricAggregationMode.per_segment, squared: bool = False,  **kwargs):
+        """Init metric.
+
+        Parameters
+        ----------
+        mode: 'macro' or 'per-segment'
+            metrics aggregation mode
+        kwargs:
+            metric's computation arguments
+        """
+        super().__init__(mode=mode, metric_fn=mse, **kwargs)
```

Comment:
Do't implement `__call__` method, you just need to pass correct `metric_fn` like this:
```
rmse = partial(mse, squared=False)
super().__init__(mode=mode, metric_fn=rmse, **kwargs)
```